hack/make/build-deb

```diff
@@ -86,6 +88,11 @@ set -e
 		cat >> "$DEST/$version/Dockerfile.build" <<-EOF
 			RUN cp -aL hack/make/.build-deb debian
 			RUN { echo '$debSource (${debVersion}-0~${suite}) $suite; urgency=low'; echo; echo '  * Version: $VERSION'; echo; echo " -- $debMaintainer  $debDate"; } > debian/changelog && cat >&2 debian/changelog
+		EOF
+		if [[ ${new_systemd[@]} =~ $version ]]; then
+			echo "RUN sed -i -- 's/#TasksMax=infinity/TasksMax=infinity/' contrib\/init\/systemd\/docker.service" >> "$DEST/$version/Dockerfile.build"
```

Comment:
But this changes of a `Dockerfie` will result of a *dirty* working directory as there will be a changes not commited. I think @thaJeztah is right on this, this probably result of a `-unsupported` build.